syntax = "proto3";

package acafela.rpc;

option java_package = "com.acafela.harmony.dirserv";

import "Common.proto";

service DirectoryService {
    rpc update(DirInfo) returns (Error);
}

message DirInfo {
    string phone_number = 1;
    string address = 2;
}
